SoftLayer/managers/hardware.py

```diff
@@ -269,6 +270,7 @@ def get_hardware(self, hardware_id, **kwargs):
                 'hourlyBillingFlag,'
                 'tagReferences[id,tag[name,id]],'
                 'networkVlans[id,vlanNumber,networkSpace],'
+                'monitoringServiceComponent,networkMonitors[queryType,lastResult,responseAction],'
```

Comment:
I think I would prefer you define this mask in the command itself, and send it in as an argument. The mask in here is getting fairly lengthy already and I would like to avoid growing it further.